aunt-leahs-functions/volunteers/index.js

```diff
@@ -94,21 +110,35 @@ module.exports = function (context, req) {
         const relationship = formInput.contactRelationship;
         const contactEmail = formInput.contactEmail;
 
-        var queryString = `INSERT INTO EmergencyContact (firstName, lastName, phoneNumber, relationship, email) \nVALUES ('${firstName}','${lastName}','${phoneNumber}','${relationship}','${contactEmail}');`
+        var queryString = 'INSERT INTO EmergencyContact (firstName, lastName, phoneNumber, relationship, email) \
+                            VALUES (@firstName, @lastName, @phoneNumber, @relationship, @contactEmail); \
+                            SELECT @@identity';
 
-        context.log(queryString);
         request = new Request(
             queryString,
-            function(err) {
+            function (err) {
                 if (err) {
                     context.log(err);
-                    context.done();
+                    context.done(err);
                 }
             });
 
-            request.on('requestCompleted', function () {
-                postVolunteers();
-            });
+        request.addParameter('firstName', TYPES.NVarChar, firstName);
+        request.addParameter('lastName', TYPES.NVarChar, lastName);
+        request.addParameter('phoneNumber', TYPES.NVarChar, phoneNumber);
+        request.addParameter('relationship', TYPES.NVarChar, relationship);
+        request.addParameter('contactEmail', TYPES.NVarChar, contactEmail);
+
+        var emergencyContactId = null;
+
+        request.on('row', function (columns) {
+            context.log('New id: %d', columns[0].value);
+            emergencyContactId = columns[0].value;
+        });
+
+        request.on('requestCompleted', function () {
+            postVolunteers(emergencyContactId);
+        });
```

Comment:
Should there be some sort of catch in case the emergency contact ID is null here? In the off chance there is some error establishing the connection